mobile/native/src/ln_dlc/mod.rs

```diff
@@ -1193,6 +1194,24 @@ pub fn is_dlc_channel_confirmed() -> Result<bool> {
     node.inner.is_dlc_channel_confirmed(&dlc_channel.channel_id)
 }
 
+pub fn estimated_fee_reserve() -> Result<Amount> {
+    let node = state::get_node();
+
+    // TODO: Here we assume that the coordinator will use the same confirmation target AND that
```

Comment:
I don't think this needs to be a TODO, it can just be a note given this function is just estimating.